receiver/kymastatsreceiver/documentation.md

```diff
@@ -0,0 +1,50 @@
+[comment]: <> (Code generated by mdatagen. DO NOT EDIT.)
+
+# kymastats
+
+## Default Metrics
+
+The following metrics are emitted by default. Each of them can be disabled by applying the following configuration:
+
+```yaml
+metrics:
+  <metric_name>:
+    enabled: false
+```
+
+### kyma.module.status.conditions
+
+The module status conditions. Possible metric values for condition status are 'True' => 1, 'False' => 0, and -1 for other status values. The metric including condition reason, status, and type as metric attribute.
```

Comment:
I don't understand this sentence:
"The metric including condition reason, status, and type as metric attribute."
What's the verb?